scripts/echo_crds_context

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [[ "${CRDS_CONTEXT}" ]]; then
+    echo $(crds list --contexts ${CRDS_CONTEXT} --resolve-contexts)
```

Comment:
This is a step backwards if you plan on supporting Windows in the future.

Reply:
Doh!  You're right.  Any suggestions on something that could be used instead?  Do I really have to write a github action to do this?  😂 

Of course CRDS and it's CLI is not Windows compatible at all right now, so that's probably the larger hurdle.  Maybe the new CLI for CRDS would actually have the above functionality as a single command?  Can only hope.

Reply:
This is a little frustrating because I thought CRDS is implemented in Python in the first place! Is there no way to call this from the Pythonic API?

Regardless, perhaps you want to look at https://docs.pytest.org/en/stable/example/simple.html#adding-info-to-test-report-header instead of creating a custom script like this. If Python API isn't an option, at least code it up such that it is a no-op if Windows is detected?

Reply:
Yes, implemented in python, but the CLI is a non-trivial map to the underlying library code.  I think I've figured it out.

And thanks for the excellent report header integration suggestion.  This makes it much slicker, and is now available to users even without using `tox`.  🎉